packages/@aws-cdk/aws-eks/lib/kubectl-handler/apply/__init__.py

```diff
@@ -40,20 +41,23 @@ def apply_handler(event, context):
     logger.info("manifest written to: %s" % manifest_file)
 
     if request_type == 'Create' or request_type == 'Update':
-        kubectl('apply', manifest_file)
+        opts = []
+        if prune_label is not None:
+            opts = ['--prune', '-l', prune_label]
+        kubectl('apply', manifest_file, *opts)
```

Comment:
Thanks for this change! We can now address this issue in similar fashion: https://github.com/aws/aws-cdk/issues/11763

Reply:
> Thanks for this change! We can now address this issue in similar fashion: https://github.com/aws/aws-cdk/issues/11763

Yes. Sounds pretty straightforward to support. Contributions are welcome!

Reply:
Yeah. Planning to contribute post this change is merged!